docs/data/date-pickers/localization/localization.md

```diff
@@ -108,7 +108,9 @@ This method can be combined with the ones shown above.
 If you pass a locale through `LocalizationProvider` or the theme, and you provide translation keys through the `localeText` prop of a picker at the same time, then only the latter translation keys will be overridden.
 
 ```tsx
-<LocalizationProvider localeText={frFR}>
+<LocalizationProvider
+  localeText={frFR.components.MuiLocalizationProvider.defaultProps.localeText}
```

Comment:
Since the purpose is about priority between the different way of customizing, we could pass an object and after the code example explaining what will be the result:

```suggestion
  localeText={{ clearButtonLabel: "Reset", todayButtonLabel: "Now" }}
```

- today button with text "Now" taken from the Localization Provider's localeText prop
- clear button with text "Vider" overridden by the Date Picker's localeText prop

Reply:
Thank you. That's a very good suggestion. 👌

Reply:
[Updated](https://github.com/mui/mui-x/pull/7102/commits/7585ec149ebc19a1a47851245eb4c9d9f51672e7) as per your suggestion. 😉